task/kaniko/0.6/kaniko.yaml

```diff
@@ -58,7 +58,7 @@ spec:
       securityContext:
         runAsUser: 0
     - name: write-url
-      image: docker.io/library/bash:5.1.4@sha256:b208215a4655538be652b2769d82e576bc4d0a2bb132144c060efc5be8c3f5d6
+      image: docker.io/library/bash:5.1.4@sha256:c523c636b722339f41b6a431b44588ab2f762c5de5ec3bd7964420ff982fb1d9
```

Comment:
@MrMYHuang, thanks for the patch. I can see that the image `gcr.io/kaniko-project/executor:v1.5.1@sha256:c6166717f7fe0b7da44908c986137ecfeab21f31ec3992f6e128fff8a94be8a5` is also multi-arch ie it supports amd64, arm64 and ppc64le. Can we edit the platforms annotation as well?
https://github.com/tektoncd/catalog/pull/1071/files#diff-dfd295d5b6103217a64c98fe1f1c91274095bbf6e55181c1d4d56cc896a58d46R12

Reply:
Done.